src/graphql/schema.js

```diff
@@ -193,7 +193,7 @@ type PaginatedTopics {
 
 
 type AccumulatedVote {
-  topicAddress: String!
+  topicAddress: String
```

Comment:
why remove the required `topicAddress`?

Reply:
for the global leaderboard, we don't have topicAddress passed in, so it will be null